h2o-algos/src/test/resources/hex/tree/h_test_input.csv

```diff
@@ -0,0 +1,91 @@
+feature0, feature1, feature2
```

Comment:
it already is but not synced yet to all testing instances so I temporarily put it here so that tests are not failing. will remove before merging / when .csv is synced.

Reply:
:+1: